CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ x.y.z Release notes (yyyy-MM-dd)
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-cocoa/issues/????), since v?.?.?)
 * None.
 
-<!-- ### Breaking Changes - ONLY INCLUDE FOR NEW MAJOR version -->
+### Breaking Changes
+* The following functionality has changed for accuracy:
+
+| Old API                                                      | New API                                                        |
+|:-------------------------------------------------------------|:---------------------------------------------------------------|
+| `Credentials(googleToken:)`                                           | `Credentials(googleAuthCode:)`                             |
```

Comment:
This seems misaligned. Also this table seems excessive for two small changes:
```md
* Change Google Credential parameter names to better reflect the required auth code:
    * `Credentials(googleToken:)` => `Credentials(googleAuthCode:)`
    * `[RLMCredentials credentialsWithGoogleToken:]` => `[RLMCredentials credentialsWithGoogleAuthCode:]`
```
Remember as well that CHANGELOG additions should be in the imperative mood (we don't do this everywhere, but we should).